py-kms/pykms_WinService.py

```diff
@@ -0,0 +1,55 @@
+# Warning: this script only for Windows 7 SP1 / Windows 2008 R2!
```

Comment:
Can you please elaborate on compatibility here? Both of these operating systems are out of support.

Reply:
> Can you please elaborate on compatibility here? Both of these operating systems are out of support.

Hello.
https://docs.microsoft.com/en-us/troubleshoot/windows-client/windows-7-eos-faq/windows-7-extended-security-updates-faq

Reply:
I try using

```
def main(self):
        self._proc = subprocess.run(self._cmd)
```
instead of
```
 def main(self):
        self._proc = subprocess.Popen(self._cmd)
        self._proc.wait()
```

but service can't be stop from services.msc console.

subprocess.run is new in Python3.